docs/articles/playbooks/Expensify-Chat-Playbook-for-Conferences.md

```diff
@@ -1,113 +1,93 @@
+As a conference organizer, you’re expected to amaze and inspire attendees. You want attendees to get to the right place on time, engage with the speakers, and create relationships with each other that last long after the conference is done. Enter Expensify Chat, a free feature that allows attendees to interact with organizers and other attendees in realtime. With Expensify Chat, you can:
 
 - Communicate logistics and key information
-- Foster conference-wide attendee networking
-- Organize discussions by topic and audience
-- Continue conversations long after the event itself 
+- Foster conference wide attendee networking
+- Organize conversations by topic and audience
+- Continue conversations long after the event itself
 - Digitize attendee social interaction
+- Create an inclusive environment for virtual attendees
 
-Sounds good? Great! In order to ensure your team, speakers, and attendees have the best experience possible, we’ve created a guide on how to use Expensify Chat at your event. 
+Sounds good? Great! In order to ensure your team, your speakers, and your attendees have the best experience possible, we’ve created a guide on how to use Expensify Chat at your event.
 
-_Let’s get started!_
+*Let’s get started!*
 
 
 ## Support
+Connect with your dedicated account manager in any new.expensify.com #admins room. Your account manager is excited to brainstorm the best ways to make the most out of your event and work through any questions you have about the setup steps below.
 
-Connect with your dedicated Expensify account manager in any new.expensify.com #admins room. Your account manager is excited to brainstorm the best ways to make the most out of your event and work through any questions you have about the setup steps below. 
-
-
-## How to set up your conference on Expensify Chat
+We also have a number of [moderation tools](https://help.expensify.com/articles/other/Everything-About-Chat#flagging-content-as-offensive) available to admins to help make sure your event is seamless, safe, and fun!
 
+## Step by step instructions for setting up your conference on Expensify Chat
 Based on our experience running conferences atop Expensify Chat, we recommend the following simple steps:
 
-### Step 1: Create your event workspace in Expensify 
-
+### Step 1: Create your event workspace in Expensify
 To create your event workspace in Expensify:
-1. In new.expensify.com: “+” > “New workspace” 
-2. Name the workspace (e.g. “ExpensiCon”)
+1. In [new.expensify.com](https://new.expensify.com): “+” > “New workspace”
+1. Name the workspace (e.g. “ExpensiCon”)
 
 ### Step 2: Set up all the Expensify Chat rooms you want to feature at your event
-
-*Protip*: Your Expensify account manager can complete this step with you. Chat them in #admins on new.expensify.com to coordinate!
+**Protip**: Your Expensify account manager can complete this step with you. Chat them in #admins on new.expensify.com to coordinate!
```

Comment:
```suggestion
**Protip**: Your Expensify account manager can complete this step with you. Chat them in #admins on [new.expensify.com](https://new.expensify.com) to coordinate!
```
NAB

NAB: Also, is "Chat them" phrased like that on purpose instead of "Chat with them"?

Reply:
This was directly from the doc. https://docs.google.com/document/d/15OHoH9i9bipRTsEBi5Pud0m3TK3-i_jhv6emBvRrZqM/edit

cc @LLPeckham